src/odemis/gui/win/acquisition.py

```diff
@@ -1311,6 +1317,164 @@ def on_acquisition_done(self, future):
+        fmt = formats[dialog.GetFilterIndex()]
+
+        for filename in dialog.GetPaths():
+            logging.debug("Current file set to %s", filename)
+            self.load_data(filename, fmt)
+
+    def load_data(self, filename: str, fmt: str = None) -> None:
+        """
+        Load the data from the file and add it to the current streams
+        :param filename: name of the file to load
+        :param fmt: format of the file to load
+        """
+        data = open_acquisition(filename, fmt)
+        self.load_streams(data)
+
+    @call_in_wx_main
+    def load_streams(self, data: List[model.DataArray]) -> None:
+        """
+        Load the data in the overview viewports
+        :param data: (list[model.DataArray]) list of data arrays to load as streams
+        """
+        # Create streams from data, add to correlation controller
+        streams = data_to_static_streams(data)
+        self.correlation_points_controller.group_add_streams(streams)
+
+        # fit to content
+        for vp in self.pnl_correlation_grid.viewports:
+            vp.canvas.fit_view_to_content()
+
+    @property
+    def correlation_points_controller(self):
+        return self._correlation_points_controller
+
+    def _create_views(self, viewports):
+        """
+        Create views depending on the actual hardware present
+        return OrderedDict: as needed for the ViewPortController
+        """
+        # TODO change SEM Overview to FIB?
+        vpv = collections.OrderedDict([
+            (viewports[0], # focused view
+             {"name": "FLM Overview",
+              "stream_classes": OpticalStream,
+              }),
+            (viewports[1],
+             {"name": "SEM Overview",
+              "stream_classes": EMStream,
+              }),])
+        return vpv
+
+    def remove_all_streams(self):
+        """
+        Remove the streams we added to the view on creation
+        Must be called in the main GUI thread
+        """
+        # Ensure we don't update the view after the window is destroyed
+        self.streambar_controller.clear()
+        gc.collect()  # To help reclaiming some memory
+
+    def terminate_listeners(self):
+        """
+        Disconnect all the connections to the streams.
+        Must be called in the main GUI thread.
+        """
+        # stop listening to events
+        self._view.stream_tree.flat.unsubscribe(self.on_streams_changed)
+        self.remove_all_streams()
+        # Set the streambar controller to None so it wouldn't be a listener to stream.remove
+        self.streambar_controller = None
+
+    def on_close(self, evt):
+        """Close event handler that executes various cleanup actions"""
+        assert self._correlation_points_controller.is_processing == False
+
+        self.EndModal(wx.ID_CANCEL)
+
+    def on_cancel(self, evt):
+        """Handle acquisition cancel button click"""
+        logging.info("Close button clicked, exiting 3DCT correlation")
```

Comment:
I don't think there is any place that should mention "3DCT" to the user (because it's really unclear what this means). I'd just say "correlation window".